skimage/draw/draw.py

```diff
@@ -56,15 +84,44 @@ def ellipse(r, c, r_radius, c_radius, shape=None):
            [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
            [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]], dtype=uint8)
 
+    >>> img = np.zeros((10, 12), dtype=np.uint8)
```

Comment:
I would remove the previous example in favor of your example, which is more general. One is enough I think and it has the advantage to keep the docstring as short as possible.